app/views/search/_filters.html.erb

```diff
@@ -1,32 +1,32 @@
 <h2 class="heading-medium"> <%= t('.filter_by') %></h2>
 
-<div class="form-group">
-  <%= label_tag 'publisher', t('.filter_by_publisher'), class: 'form-label' %>
-  <%= select_tag('filters[publisher]', options_for_select(dataset_publishers_for_select, selected_publisher), id: 'publisher', class: 'form-control dgu-filters__filter', include_blank: true) %>
+<div class="govuk-form-group">
+  <%= label_tag 'publisher', t('.filter_by_publisher'), class: 'govuk-label' %>
+  <%= select_tag('filters[publisher]', options_for_select(dataset_publishers_for_select, selected_publisher), id: 'publisher', class: 'govuk-select dgu-filters__filter', include_blank: true) %>
```

Comment:
This looks like the markup for the select component, is that right? Why is it written out rather than calling the component? e.g. 

```
<%= render "govuk_publishing_components/components/select", {
  id: "dropdown1",
  label: "My Dropdown",
  options: [
    {
      text: "Option one",
      value: "option1"
    },
  ]
} %>

Reply:
This is an accessible autocomplete instance. I wasn't sure if it was possible to include the select component here and for the AA to still work. I will try this out and get back to you.

Reply:
It probably isn't, so this is probably fine as it is. It wasn't obvious that an AA is applied here, is that done through a specific class? I think it might be worth making this more obvious, perhaps by using a specific `js-` prefixed class for example.

Reply:
I haven't been able to get this working with the select govpubcomp but I've added a js class to label it so it's clear that it's being used for AA.